### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by chidiwilliams** [chidiwilliams/buzz#688] [Announcements]
## What's Changed
* Update language translations
* Update docs
* Publish to PyPI
* Upgrade to Whisper v3 in https://github.com/chidiwilliams/buzz/pull/626
* Fix OpenAI API transcriber audio limits
* Add folder watch
* Update Linux Snap build
* Add URL imports
* Fix CLI generating blank filenames
* Save transcriptions to SQLite
* Fix Linux Keyring exceptions in https://github.com/chidiwilliams/buzz/pull/685
* Add transcript text viewer in https://github.com/chidiwilliams/buzz/pull/686

## New Contributors
* @albanobattistella made their first contribution in https://github.com/chidiwilliams/buzz/pull/592
* @fitojb made their first contribution in https://github.com/chidiwilliams/buzz/pull/628
* @Sebek05 made their first contribution in https://github.com/chidiwilliams/buzz/pull/658

**Full Changelog**: https://github.com/chidiwilliams/buzz/compare/v0.8.4...v0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0'>v0.9.0</a>.</em>

**Comment by nitchevcasseus** [comment]
Awesome updates: If you can create something g like Meta Seamless Streaming. I honestly thing that would be game changing. A lot of people are looking for Real Time Speech to Speech Translation. 

I myself would be more than happy to pay for a solution like that.

**Reply by Jackvrs** [reply]
Hi

[v0.8.4](https://github.com/chidiwilliams/buzz/releases/tag/v0.8.4) its typed good 

**Latest Version** 
[v0.9.0](https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0)
Upgrade to Whisper v3 lot of Words is skip 
if audio is good its typed good . if any background noise its skiped or typed.

Paragraph 12. The report on file supports the history. Paragraph 13. The report on file supports the history. Paragraph 14. The report on file supports the history. Paragraph 15. The report on file supports the history.
Paragraph 16. The report on file supports the history. Paragraph 17. The report on file supports the history. Paragraph 18. The report on file supports the history. Paragraph 19. The report on file supports the history. Paragraph 20. The report on file supports the history. Paragraph 21. The report on file supports the history. Paragraph 22. The report on file supports the history. Paragraph 23. The report on file supports the history. 

 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25,
 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42,
 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 60,
 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60,
 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60,
 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60,
 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60, 60,
 
================
I am using medical Transcription voices 

Buzz all versions below type of medical Words in Error 
Hematocesia = hematochezia 
angelfisher – anal fissure 
if its possible to change in the next update
and one more think 
pls add the options for paragraph marks and  Replace Single space to double space after period.

**Reply by PaulANormanNZ** [reply]
Hi Jackvrs,

I _directly_ explored the command line of Whisper-Faster _r192.3 CLI
[github / Purfview / whisper-standalone-win](https://github.com/Purfview/whisper-standalone-win)

and it warned me (after it had finished processing! :-) 
that I had elected to use the large language model version 3:
 faster-whisper-large-v3 
 
and that it might be inaccurate, and that I could better choose to revert back to 
version 2: faster-whisper-large-v2
— for better results.

So the issue may not be in the **Buzz 0.90** update itself here, 
but with any subsidiary language model update which has been automatically incorporated?

Kind regards,
Paul

**Comment by Sircam19** [comment]
As suggested by Candideau to a similar question now posted here.  When might we expect to see version 0.9.0 appear in the Mac App Store? It hasn't appeared via the Homebrew method as per the explanation cited by Chidi but no information on the other installation method when on a Mac. Regards. 

![CleanShot 2024-03-18 at 23 34 12](https://github.com/chidiwilliams/buzz/assets/4651001/e8f01ea2-3732-4fa5-b66b-a8bc952d5612)

**Comment by chidiwilliams** [comment]
The open-source and App Store versions have different release cycles and
versions. The features in 0.9.0 are already available in the latest App
Store version.

**Reply by Sircam19** [reply]
Thanks so much for clarifying. That's perfect. Downloading again now. Cheers.

**Comment by FullThrottle83** [comment]
Will there be an updated v.0.9 windows .exe also? :)

**Reply by chidiwilliams** [reply]
There will. Once I get a code signing certificate, hopefully in a few days.

**Reply by FullThrottle83** [reply]
Thanks alot for your hard work!!

**Reply by 111ewe111** [reply]
This would be great, thank you!  Can this be paired with a "speaker diarization" engine like Picovoice's Falcon to pinpoint individual speakers?  Would be cool if you added this level of notation, too.

**Reply by lepopetre** [reply]
[chidiwilliams](https://github.com/chidiwilliams) thank's a lot for your purpose it's fully efficient transcription from french !  

I'd like to know if you want it to be free for use how can we support ??

**Reply by tim3r** [reply]
Looking forward to the exe file. Thank you very much!

**Reply by obaidshah345** [reply]
Hey @chidiwilliams great effort mate! Eagerly waiting for the exe file though.

**Reply by MrFutureV** [reply]
bro still waiting

**Reply by chidiwilliams** [reply]
Updated now. Please check and let me know https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0

**Comment by Faoineag** [comment]
Good evening, forgive the question, I have version 0.8.4 and I saw that 0.9 is available, where can I find the exe file? Thank you

**Reply by DirkSchlossmacher** [reply]
What is actually required to generate an exe with the source?

**Reply by chidiwilliams** [reply]
Available now: https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0.

There was an exe at first, but Windows was flagging as it wasn't signed. Fixed now.

**Comment by Faoineag** [comment]
I asked about the exe because I don't know how else to update the version I have with the latest one, I'm not that expert and I have no idea how to use the source file. Thank you.

Sabrina

 

Da: DirkSchlossmacher ***@***.***) ***@***.***> 
Inviato: giovedì 28 marzo 2024 09:27
A: chidiwilliams/buzz ***@***.***>
Cc: Faoineag ***@***.***>; Manual ***@***.***>
Oggetto: Re: [chidiwilliams/buzz] v0.9.0 (Discussion #688)

 

What is actually required to generate an exe with the source?

**Reply by chidiwilliams** [reply]
.exe file is available now https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0

**Comment by PaulANormanNZ** [comment]
If we install an .exe from here. we are going to get security prompts during install with or without  your haviong a security certificate I think—I just put 0.84 windows exe on from here under 0.8's assets (until the 0.9 exe is available), and that is what happened anyway, even though it is signed? 

I am guessing that the certificate will only help with you putting it up on the Microsift App store?

**Reply by chidiwilliams** [reply]
The signed exe is available now: https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0.

It shouldn't have the security prompts during install but let me know if you see differently.

**Reply by mazhengeod** [reply]
thank u ！

**Reply by PaulANormanNZ** [reply]
Hi,

Thanks Great!

I think that if activated, Windows Defender might always prompt during install from the download from here (Github), and not the Windows App Store?

Although you would think that a current Windows Certificate should bypass this?
For a first time user it could seem like a pretty heavy message!

![0 9_setup_Windows_Defender_message](https://github.com/chidiwilliams/buzz/assets/457267/a6bcf987-d391-442a-830f-e85a0b05812d)

**Comment by mktnag** [comment]
Hi @chidiwilliams thanks a ton for Buzz—great and useful! Noticed the App Store version doesn’t support macOS Monterey, and I'm unable to upgrade my Mac to the newer Ventura OS version. Is there a plan to release a .dmg compatible with Monterey?

**Reply by chidiwilliams** [reply]
Hi, that's interesting. I actually didn't realise that was the case. I'll see if there's any Ventura-specific code I'm depending on and downgrade in the next version if there isn't any.

**Comment by mazhengeod** [comment]
help!  I can not find the exe file on this page,I am a Windows user,and I really need these tools for work.

**Reply by chidiwilliams** [reply]
Updated now https://github.com/chidiwilliams/buzz/releases/tag/v0.9.0

**Comment by mazhengeod** [comment]
I already downloaded those ZIP files on this page, But it seems like panch original code, it too complex for me to run it

**Reply by raivisdejus** [reply]
Executable versions for Windows and Mac you can find in the release section https://github.com/chidiwilliams/buzz/releases

Linux version here https://snapcraft.io/buzz

Python PiPy version is here https://pypi.org/project/buzz-captions/

**Comment by tim3r** [comment]
Thanks for .exe! 🥇

**Comment by lepopetre** [comment]
I downloaded and installed.

1h ago

Was a safety management prompt

I tried the folder tools ... Let you know

But after the lmatch

Le mer. 10 avr. 2024, 21:13, Chidi Williams ***@***.***> a
écrit :

**Comment by evantianzg** [comment]
Hi @chidiwilliams, thanks for the updates. 

Will there be a dmg file for Mac OS? I am currently using 0.8.4 version and I found the live recording does not work on my laptop (Mac OS Sonoma 14.1.2). I was wondering if 0.9 would solve this problem?

**Reply by raivisdejus** [reply]
Latest versions are available in release section https://github.com/chidiwilliams/buzz/releases

**Comment by exkson** [comment]
Is a documentation available to build from source for Linux ?

**Reply by raivisdejus** [reply]
See notes here https://github.com/chidiwilliams/buzz/blob/main/CONTRIBUTING.md

**Comment by kiya52** [comment]
Dear chidiwilliams,

I am kiya. Very thanks for your update v0.9.0 of Buzz.
I have already install in a windows 10 laptop( CPU-i7 10th, 8G RAM, with Nvidia-Geforce GTX1650. (acer Nitro 5) )
This software can work in whisper model( base ,small, medium) by transcibe chinese. (I am from Taiwan.)
When I transcate an about 4mins's mp3-file, this computer needs about 1mins by base model, 2mins by small model, 8mins by medium model to excecise voice to text.

But it can't not use the model by fast whisper model to transcate the same mp3 file.
The software status shows "fail (Unknown errer)".

I have some questions as follows:
1. How can I use other models? should I download or install other model?
2. Can I set Buzz using GPU or CUDA to do voice to text?

Thanks for your kindly replying.
sincerely 

kiya chang

**Reply by raivisdejus** [reply]
One thin you can try is custom OpenAI API, enter it in the preferences and the select "OpenAI Whisper API" when transcribing. Notes on API endpoints available are here https://github.com/chidiwilliams/buzz/discussions/827

Whisper, Faster Whisper and Hugging Face versions should all use CUDA GPU if it is installed. https://developer.nvidia.com/cuda-downloads.
Seems you have 4GB of VRAM, try Faster Whisper, it needs less VRAM. It may work, but may also fail.

**Comment by kiya52** [comment]
Dear chidiwilliams,

Another questions is:
1.  Can I add a "usually used words" to let the result of voice to text better than now?

**Reply by raivisdejus** [reply]
To improve quality of speech recognition you can donate your voice to https://commonvoice.mozilla.org
Whisper models are trained on this data, and eventually will get better if more data is available.

**Comment by kiya52** [comment]
Dear chidiwillams, 
I got your new vision of buzz v1.0.0， Thanks for your update.

I will try it.
